moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -303,8 +303,15 @@ void RobotStateDisplay::newRobotStateCallback(const moveit_msgs::DisplayRobotSta
     return;
   if (!robot_state_)
     robot_state_.reset(new robot_state::RobotState(robot_model_));
-  // possibly use TF to construct a robot_state::Transforms object to pass in to the conversion functio?
-  robot_state::robotStateMsgToRobotState(state_msg->state, *robot_state_);
+  // possibly use TF to construct a robot_state::Transforms object to pass in to the conversion function?
+  try
+  {
+    robot_state::robotStateMsgToRobotState(state_msg->state, *robot_state_);
+  }
+  catch (const moveit::Exception& e)
+  {
+    ROS_WARN_ONCE("moveit::Exception %s", e.what());
```

Comment:
Please use `setStatus(rviz::StatusProperty::Error, "RobotState", "[...]");`, so the problem shows in RViz too.